#! /bin/sh /usr/share/dpatch/dpatch-run
## 25-no-special-build-dir-for-d64.dpatch by Tiago Saboga <tiagosaboga@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Do not build files for 64 bit platforms in a separate dir, as
## DP: this would mean detecting it again in package building scripts.

@DPATCH@
diff -urNad libjna-java-3.0.4~/build.xml libjna-java-3.0.4/build.xml
--- libjna-java-3.0.4~/build.xml	2008-07-25 00:08:43.000000000 -0300
+++ libjna-java-3.0.4/build.xml	2008-07-25 00:11:48.000000000 -0300
@@ -76,7 +76,7 @@
         <os arch="amd64"/>
       </or>
     </condition>
-    <property name="build" value="build${vm.arch}"/>
+    <property name="build" value="build"/>
     <property name="classes" location="${build}/classes"/>
     <property name="eclipse.classes" location="build.eclipse/classes"/>
     <property name="test.classes" location="${build}/test-classes"/>
@@ -396,16 +396,12 @@
     <condition property="vmopt.arch" value="-d64" else="-Dignore">
       <equals arg1="${vm.arch}" arg2="-d64" trim="true"/>
     </condition>
-    <condition property="dir.build.suffix" value="-d64" else="">
-      <equals arg1="${vm.arch}" arg2="-d64" trim="true"/>
-    </condition>
     <property name="tests.platform" value=""/>
     <junit printsummary="yes" fork="${test.fork}" failureproperty="testfailure">
       <jvmarg value="-Djna.library.path=${build.native}"/>
       <jvmarg value="-Djava.library.path=${build.native}:${java.library.path}"/>
       <!-- Avoid VM crashes, if possible -->
       <jvmarg value="-Djna.protected=true"/>
-      <jvmarg value="-Ddir.build.suffix=${dir.build.suffix}"/>
       <jvmarg value="${vmopt.arch}"/>
       <classpath><path refid="test.runpath"/></classpath>
       <formatter type="xml"/>
